region/crateria/east/The Moat.json

```diff
@@ -563,6 +555,34 @@
         "Morph just before hitting the ceiling, in order to extend the jump horizontally."
       ]
     },
+    {
+      "link": [1, 2],
+      "name": "Come in Getting Blue Speed, Leave With Temporary Blue (Spring Ball Bounce)",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 2,
+          "openEnd": 0,
+          "steepDownTiles": 1,
+          "minExtraRunSpeed": "$0.8"
+        }
+      },
+      "requires": [
+        "canTrickySpringBallBounce",
+        "canChainTemporaryBlue"
+      ],
+      "exitCondition": {
+        "leaveWithTemporaryBlue": {}
+      },
+      "unlocksDoors": [{"types": ["ammo"], "requires": []}],
+      "collectsItems": [3],
+      "flashSuitChecked": true,
+      "note": [
+        "Enter with blue speed and bounce on the item pedestal to make it across, then unmorph into temporary blue."
```

Comment:
```suggestion
        "Enter with blue speed and bounce on the item pedestal to make it across, then unmorph into temporary blue.",
        "Morph on the left side of the pedestal so that there is enough space to use a controlled SpringBall bounce."
```